app/lib/scihist_digicoll/env.rb

```diff
@@ -383,6 +383,8 @@ def self.dzi_backup_bucket
 
     # From address:
     define_key :no_reply_email_address, default: "no-reply@sciencehistory.org"
+    define_key :digital_collections_email_address, default: "digital@sciencehistory.org"
```

Comment:
Hi, is there an intended difference in meaning between this key and `:digital_email_address` below, pointing to same email address? Or did you just not notice that one?

Reply:
Didn't notice it! Thanks for catching this.